dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/listener/CompressionSMSListener.java

```diff
@@ -245,6 +249,13 @@ private SMSMetadata getMetadata( Date lastSyncDate )
         meta.trackedEntityAttributes = getTypeUidsBefore( TrackedEntityAttribute.class, lastSyncDate );
         meta.programs = getTypeUidsBefore( Program.class, lastSyncDate );
         meta.organisationUnits = getTypeUidsBefore( OrganisationUnit.class, lastSyncDate );
+        meta.programStages = getTypeUidsBefore( ProgramStage.class, lastSyncDate );
```

Comment:
@JasperTimm the SDK is not downloading this metadata. Should we do it? Also, I would say that trackedEntityInstances, enrollments, events and relationships are data. In a typical use case the android app would not have all this data because it could be huge

Reply:
@vgarciabnz yeah that's a good point, I was going to discuss that with you guys, I sort of came to the same conclusion (that TEIs, enrolments and events are more so data). In that case I'll remove the handling for it from the compression library.
Also yes, I think it makes sense for the SDK to download all other metadata. ProgramStages is quite small I imagine.

Reply:
Yes, i think it makes sense to include programStages, relationshipTypes and dataSets.